### DISCUSSION TITLE
How to Programmatically Add an Item to a Specific Index in a DropZone Without Drag and  Drop Event

### DISCUSSION THREAD

**Discussion by MarkizaZ** [Megabit/Blazorise#5726] [Q&A]
I'm working on application that uses drag-and-drop functionality to manage a list of  items within a DropZone with reorder allowed. I have a requirement where I need to add a new item to a specific index in the drop zone directly, bypassing the usual drag-and-drop event. Is it possible?

**Comment by alen-resser** [comment]
I also have this problem and it would be super useful if i could read the index position of an element
